MHVLib/Connection/MHVSodaConnectionProtocol.h

```diff
@@ -0,0 +1,41 @@
+//
+//  MHVSodaConnectionProtocol.h
+//  MHVLib
+//
+// Copyright (c) 2017 Microsoft Corporation. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import <Foundation/Foundation.h>
+#import "MHVConnectionProtocol.h"
+
+/**
+ A client connection to HealthVault
+ */
+@protocol MHVSodaConnectionProtocol <MHVConnectionProtocol>
+
+/**
+ Prompt the user to authorize additional records for use in this app.
```

Comment:
Will this show a webview?  Should have an optional view controller as a parameter if so